app/models/claim/transfer_brain_data_item.rb

```diff
@@ -1,23 +1,18 @@
 module Claim
   class TransferBrainDataItem
-    attr_reader :litigator_type, :elected_case, :transfer_stage_id, :case_conclusion_id,
-                :allocation_type, :validity, :transfer_fee_full_name
+    # FIXME: does not look like these are used anywhere
+    # attr_reader :litigator_type, :elected_case, :transfer_stage_id, :case_conclusion_id,
+    # :allocation_type, :validity, :transfer_fee_full_name, :bill_scenario
```

Comment:
Can this be deleted if they aren't used?

Reply:
hopefully, yes. I am going to do more of a refactor on this file/functionality anyway so will do it then.